.vib/drupal/goss/vars.yaml

```diff
@@ -0,0 +1,16 @@
+drupalUsername: test_user
+drupalPassword: ComplicatedPassword123!4
```

Comment:
You should not just paste the used `runtime_params` here, as there are parameters you won't be checking in goss. Please remove those unrelated to your tests.

Additionally, `bitnami/drupal` automounts a service account, please add the related tests as done for other assets:

https://github.com/bitnami/charts/blob/c6aa735f9d8e6352d951bdfc92befacff69ebe0f/.vib/cassandra/goss/goss.yaml#L14